client/blocks/product-selector/index.jsx

```diff
@@ -145,7 +154,7 @@ export class ProductSelector extends Component {
 					title={ product.title }
 					billingTimeFrame={ this.getBillingTimeFrameLabel() }
 					fullPrice={ productObject.cost }
-					description={ <p>{ product.description }</p> }
```

Comment:
What's the reason to drop the `<p>` tag here? The description is rendered [without any wrapper](https://github.com/Automattic/wp-calypso/blob/f7d096b6e9a7147e3326ecd15cf8846b9b34a2e6/client/components/product-card/index.jsx#L69) inside the `<ProductCard />` and that's where the paragraph tag comes from here.

Reply:
Never mind, @enejb! Just realized the `<p>` tag is added in the parent component.

Reply:
I was getting an error such as 
```
Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    in p
    in p (created by ProductSelector)
    in div (created by ProductCard)
    in div (created by Card)
    in Card (created by ProductCard)
    in ProductCard (created by ProductSelector)
    in div (created by ProductSelector)
    in ProductSelector (created by Context.Consumer)
    in WithLocalizedMoment(ProductSelector) (created by Localized(WithLocalizedMoment(ProductSelector)))
    in Localized(WithLocalizedMoment(ProductSelector)) (created by Connect(Localized(WithLocalizedMoment(ProductSelector))))
    in Connect(Localized(WithLocalizedMoment(ProductSelector))) (created by PlansFeaturesMain)
    in div (created by PlansFeaturesMain)
    in div (created by PlansFeaturesMain)
    in PlansFeaturesMain (created by Localized(PlansFeaturesMain))
    in Localized(PlansFeaturesMain) (created by Connect(Localized(PlansFeaturesMain)))
    in Connect(Localized(PlansFeaturesMain)) (created by Plans)
    in div (created by Plans)
    in main (created by Main)
    in Main (created by Plans)
    in div (created by Plans)
    in Plans (created by WithTrackingTool( Plans ))
```